include/aspect/simulator_signals.h

```diff
@@ -70,6 +70,20 @@ namespace aspect
                                   ConstraintMatrix &)>  post_constraints_creation;
 
     /**
+    * A signal that is called before setting up the degrees of
+    * freedom for the simulator. This allows for editing of the parameters
+    * struct  on the fly (such as changing boundary conditions) to give
+    * Aspect different behaviour in mid-run than it otherwise would have.
```

Comment:
You're American -- I'm surprised you spell behavior as you do :-)

There's also a double space on the line before.